FrontEnd/src/components/Login.tsx

```diff
@@ -3,8 +3,27 @@ import { Typography } from '@mui/material';
 import '../CSS/Login.css';
 import image from '../login-image.png';
 import Link from '@mui/material/Link';
+import GetUser from './GetUser';
+import { useNavigate } from 'react-router-dom';
 
 export default function Login() {
+  let navigate = useNavigate();
+  interface clientPrincipal {
```

Comment:
You can export this interface from `GetUser.tsx` and import here and elsewhere to avoid repeating the definition of this object.

Reply:
Thank you @OmarShehata ! I thought about it and was going to google if interfaces in Typescript can be reused. Will do the import asap.